src/Arcus.Templates.WebApi/appsettings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I presume this is the file that is being included if the flag is turned on?

If so, shouldn't we inject this test content since others might not want to have this or not?

Reply:
It's not test content; it's required when the certificate authentication is enabled.
We have included a configuration key in the `Startup.cs` that would reflect to this structure.

We could inject both but not sure if that is really something that would make it more user-friendly.

Reply:
Yeah, that was something I would change in an upcoming PR; when we introduce the `--include-appsettings` project options. In the current state; that's the simplest thing for it to work.

Reply:
Sounds good to me, tracking in https://github.com/arcus-azure/arcus.templates/issues/64